src/resolver/mixins.cc

```diff
@@ -242,88 +353,42 @@ EnvironmentMixin::addFloxDirectoryOption( argparse::ArgumentParser & parser )
       } );
 }
 
+/* ---------------- EnvironmentMixin init overrides ------------------------- */
 
-/* -------------------------------------------------------------------------- */
-
-void
+GlobalManifest
 GAEnvironmentMixin::initGlobalManifest( GlobalManifestRaw manifestRaw )
 {
   if ( this->gaRegistry )
     {
       (void) static_cast<GlobalManifestRawGA>( manifestRaw );
+
       if ( ! manifestRaw.registry.has_value() )
         {
           manifestRaw.registry = getGARegistry();
         }
     }
-  this->EnvironmentMixin::initGlobalManifest( std::move( manifestRaw ) );
+  return this->EnvironmentMixin::initGlobalManifest( manifestRaw );
 }
 
-
 /* -------------------------------------------------------------------------- */
 
-void
-GAEnvironmentMixin::initGlobalManifestPath( std::filesystem::path path )
-{
-  if ( this->gaRegistry )
-    {
-      auto manifestRawGA = readManifestFromPath<GlobalManifestRawGA>( path );
-      auto manifestRaw   = static_cast<GlobalManifestRaw>( manifestRawGA );
-      this->setGlobalManifestPath( std::move( path ) );
-      this->EnvironmentMixin::initGlobalManifest( std::move( manifestRaw ) );
-    }
-  else { this->EnvironmentMixin::initGlobalManifestPath( std::move( path ) ); }
-}
-
-
-/* -------------------------------------------------------------------------- */
-
-void
+EnvironmentManifest
 GAEnvironmentMixin::initManifest( ManifestRaw manifestRaw )
 {
   if ( this->gaRegistry )
     {
       (void) static_cast<ManifestRawGA>( manifestRaw );
```

Comment:
non-blocking note:

This type of expression `(void) EXPR` is how we _explicitly discard_ if we _really_ want to ignore a `[[nodiscard]]`.

This is basically only useful for tests; but wanted to highlight it since it's useful.